pkg/action/executor/move_util.go

```diff
@@ -128,18 +129,23 @@ type moveHelper struct {
 	emap    *actionUtil.ExpirationMap
 	key     string
 	version int64
+
+	//stop Renewing
+	stop       chan struct{}
+	isRenewing bool
 }
 
-func NewMoveHelper(client *kclient.Clientset, nameSpace, name, kind, parentName, noneScheduler string, highver bool) (*moveHelper, error) {
+func NewMoveHelper(client *kclient.Clientset, nameSpace, name, kind, parentName, noneScheduler string, highver bool) (*schedulerHelper, error) {
```

Comment:
if the type has been changed from `moveHelper` to `schedulerHelper`, so please make the function name consistent, change `NewMoveHelper` to `NewScheudler` helper. BTW, I think `reschedulerHelper` makes more sense.

Reply:
It is a tool to manipulate the original scheduler of pod: getScheduler/updateScheduler, not really rescheduling a pod.